src/node/rpc/memberfrontend.h

```diff
@@ -104,8 +105,9 @@ namespace ccf
            auto new_member_id =
              g.add_member(parsed.cert, parsed.keyshare, MemberStatus::ACCEPTED);
 
-           auto ack = tx.get_view(this->network.member_acks);
-           ack->put(new_member_id, {rng->random(SIZE_NONCE)});
+           auto [ma_view, sig_view] =
+             tx.get_view(this->network.member_acks, this->network.signatures);
+           ma_view->put(new_member_id, {{}, sig_view->get(0)->root});
```

Comment:
If I'm reading this correctly, this is also clearing the previous signature? I think we should leave it intact if it exists?

Reply:
Ignore this, I wasn't reading it correctly - this is during member creation, so we're not overwriting an existing signature.